build/projects/AI Test Toolkit/.AL-Go/settings.json

```
@@ -0,0 +1,17 @@
+{
+    "projectName": "AI Test Toolkit",
+    "appFolders":  [
+        "../../../src/Tools/AI Test Toolkit/App"
```

```suggestion
        "../../../src/Tools/AI Test Toolkit"
```

Currently the AI test toolkit isn't being built in thr PR Build 
`Warning: No apps found in appFolders in .AL-Go\settings.json`

I suspect this is the reason